app/views/play/CampaignView.coffee

```diff
@@ -622,13 +638,13 @@ module.exports = class CampaignView extends RootView
       level.locked = true if level.requiresSubscription and @requiresSubscription and me.get('hourOfCode')
       level.locked = false if @levelStatusMap[level.slug] in ['started', 'complete']
       level.locked = false if @editorMode
-      level.locked = false if not @course? and @campaign?.get('name') in ['Auditions', 'Intro']
+      level.locked = false if not @isClassroom() and @campaign?.get('name') in ['Auditions', 'Intro']
       level.locked = false if me.isInGodMode()
       level.disabled = true if level.adminOnly and @levelStatusMap[level.slug] not in ['started', 'complete']
       level.disabled = false if me.isInGodMode()
 
       level.color = 'rgb(255, 80, 60)'
-      unless @course? or @campaign?.get('type') is 'hoc'
+      unless @isClassroom() or @campaign?.get('type') is 'hoc'
```

Comment:
Shouldn't we remove this `@isClassroom()` check since we're returning at the top of function if it's true?